spec/redis-test.conf

```diff
@@ -206,21 +206,21 @@ appendfsync everysec
 # To enable VM just set 'vm-enabled' to yes, and set the following three
 # VM parameters accordingly to your needs.
 
-vm-enabled no
+# vm-enabled no
```

Comment:
These options are no longer available on the version of redis we are using. I've commented them out so that we can still run the specs as necessary (although there is a lot of deprecation warnings!)